letsencrypt-apache/letsencrypt_apache/augeas_configurator.py

```diff
@@ -30,7 +33,8 @@ def __init__(self, *args, **kwargs):
         # Set Augeas flags to not save backup (we do it ourselves)
         # Set Augeas to not load anything by default
         my_flags = augeas.Augeas.NONE | augeas.Augeas.NO_MODL_AUTOLOAD
-        self.aug = augeas.Augeas(flags=my_flags)
+        my_dir = os.path.dirname(constants.AUGEAS_HTTPD_LENS)
```

Comment:
I'm not sure if it's safe to call `dirname` on the `resource_filename` result... `resource_filename` might create a temporary file (copy it from the distribution to some location on the disk), especially when `zip_safe == True`